pocs/utils/images/__init__.py

```diff
@@ -101,7 +101,10 @@ def _make_pretty_from_fits(fname, **kwargs):
     norm = ImageNormalize(interval=PercentileInterval(percent_value), stretch=LogStretch())
     wcs = WCS(fname)
 
-    plt.figure(figsize=(10, 8), dpi=150)
+    figsize = (10, 8)
+    dpi = 150
```

Comment:
This edit changes nothing if you're still setting the values right here in the code. Put them in the function definition, e.g. replace `def _make_pretty_from_fits(fname, **kwargs):` with `def _make_pretty_from_fits(fname, figsize=(10, 8), dpi=150, **kwargs):` and take these two lines out.

Reply:
That's what I was confused, but now I know what you mean.